models/pydantic/models.py

```diff
@@ -15,3 +17,12 @@ class AnimalResponse(BaseModel):
     animal_type: str
     name: str
     birth_date: date
+    breed: str
+    photo: str
+
+    @computed_field
+    @property
+    def age(self) -> str:
+        years = date.today().year - self.birth_date.year
```

Comment:
What if birthday is 01.12.2022 and today is 24.10.2023?

Reply:
Updated